src/QMCWaveFunctions/WaveFunctionComponent.h

```diff
@@ -250,6 +250,11 @@ struct WaveFunctionComponent: public QMCTraits
     return GradType();
   }
 
+  /** For given particle set configuration, initialize whatever intermediate objects are required
+  * for force evaluation (preferably fast).  Default does nothing.*/ 
+  virtual void prepareIonDerivs()
```

Comment:
Resizing is only needed locally. Please remove all prepareIonDerivs but DiracDeterminant.